wandb/integration/openai/fine_tuning.py

```diff
@@ -228,7 +229,8 @@ def _log_fine_tune(
         # check results are present
         try:
             results_id = fine_tune.result_files[0]
-            results = cls.openai_client.files.content(file_id=results_id).text
+            encoded_results = cls.openai_client.files.content(file_id=results_id).read()
```

Comment:
does this depend on the version of their sdk? if so, would this change break the earlier versions? should you update the minimum required version at the top of this file?

Reply:
More than their sdk I think it depends on the fine-tune jobs created now. I just tried an old fine-tune job id from December 2023 and it didn't work with the changes made. It worked fine with the a newer job id (Aug, 24).

Let me check if it breaks older versions of their sdk. If it does, I will change the minimum version to a later version - given their library is not stable, backward compatibility is not important imo.

Reply:
tested with an older version (1.12.0 vs 1.40.0). The changes worked for new fine-tune job ids vs old ones. I would still move the minimum version to 1.12.0.

Reply:
i see. please do update the minimum version.
for old vs new job - how about a try/except so that it works with both?